### DISCUSSION TITLE
How to trigger rebuild in vercel by webhook after I edit the admin ui?

### DISCUSSION THREAD

**Discussion by summerisgood1** [keystonejs/keystone#8807] [Questions]
How to trigger rebuild in vercel by webhook after I edit the admin ui?
Since, My website is static site generation

**Comment by PhilBookst** [comment]
Do you mean you edit an item in the admin ui? Like for example a post or whatever - if so then you can use the [afterOperation](https://keystonejs.com/docs/config/hooks#after-operation) list hook to make a request to your webhook. 

I've used this with next.js and their [on demand revalidation](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#on-demand-revalidation)
